repo-projects/docker-creds-provider.proj

```diff
@@ -0,0 +1,39 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <PackagesOutput>$(ProjectDirectory)/src/Valleysoft.DockerCredsProvider/bin/$(Configuration)/</PackagesOutput>
+    <RepoApiImplemented>false</RepoApiImplemented>
```

Comment:
@mthalman, Can you apply the cleanup from https://github.com/dotnet/source-build-externals/pull/115 to this PR?